runtime-manager/modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -53,393 +58,167 @@ Icons appear at the top of each page in the Runtime Manager documentation, indic
 |Anypoint Runtime Fabric
 |image:logo-rtf-active.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
 |image:logo-rtf-disabled.png[xref="deployment-strategies.adoc#anypoint-runtime-fabric-deployments", title="Runtime Fabric"]
-
 |===
 
+== Runtime Manager Features
 
-For an overview of the Runtime Manager features available with each deployment option, see:
+For an overview of the Runtime Manager features available with all deployment options, see:
 
-* <<CloudHub Deployments,CloudHub>>
-* <<Hybrid Deployments,Hybrid>>
-* <<Anypoint Platform PCE Deployments,Anypoint Platform PCE>>
-* <<Anypoint Runtime Fabric Deployments,Anypoint Runtime Fabric>>
+* <<High Availability>>
+* <<JDK Versions>>
+* <<Load Balancing>>
+* <<Logging>>
+* <<Monitoring>>
+* <<Object Store>>
+* <<Scheduling>>
+* <<Security Updates>>
+* <<Shared Resource Support>>
+* <<Troubleshooting>>
 
 
+=== High Availability
 
-== CloudHub Deployments
+[%header,cols="15,50,20"]
+|===
+|Environment | Implementation      | Reference
+|CloudHub 2.0 | Via clustering with two or more replicas, and Anypoint VPN. |   xref:cloudhub-2::ch2-clustering.adoc[Clustering], xref:cloudhub-2::ps-vpn-about.adoc#vpn-ha[VPN High Availability]. 
```

Comment:
```suggestion
|CloudHub 2.0 | Via clustering with two or more replicas and Anypoint VPN. |   xref:cloudhub-2::ch2-clustering.adoc[Clustering], xref:cloudhub-2::ps-vpn-about.adoc#vpn-ha[VPN High Availability]
```

Reply:
Check the width of the first column to see if you can keep the CH entries from wrapping - it'll make it easier to scan.
![image](https://github.com/mulesoft/docs-runtime-manager/assets/19573371/a989d008-093d-4b3a-ad7a-4501869556e9)